components/tree/TreeNode.razor.cs

```diff
@@ -337,10 +337,39 @@ public bool IsLeaf
         /// Expand the node
         /// </summary>
         /// <param name="expanded"></param>
-        public void Expand(bool expanded)
+        public async Task Expand(bool expanded)
         {
             if (Expanded == expanded) return;
             Expanded = expanded;
+
+            await TreeComponent?.OnNodeExpand(this, Expanded, new MouseEventArgs());
+        }
+
+        /// <summary>
+        /// Expand all child nodes
+        /// </summary>
+        public async Task ExpandAll()
+        {
+            await SwitchAllNodes(this, true);
+        }
+
+        /// <summary>
+        /// Collapse all child nodes
+        /// </summary>
+        public async Task CollapseAll()
```

Comment:
```suggestion
        internal async Task CollapseAll()
```

Reply:
if to internal
_tree.FindFirstOrDefaultNode(n => n.Title == "C").CollapseAll();
not work

Reply:
If you need to expand a node individually, I think you should set up a cleaner API.
How about：
```
_tree.ExpandAll("node-key")
```